examples/simple/lint/L-CCVE1-bad.output

```diff
@@ -0,0 +1,5 @@
+Warning: L-CCVE1
+Description: unsigned integer overflow.
```

Comment:
this and the one @grosu mention should be capitalized. I will make a PR to handle the code change.